choria.go

```diff
@@ -87,10 +87,14 @@ type ChoriaPluginConfig struct {
 	RemoteSignerURL              string   `confkey:"plugin.choria.security.request_signer.url"`
 
 	// file security
-	FileSecurityCertificate string `confkey:"plugin.security.file.certificate" type:"path_string"`
-	FileSecurityKey         string `confkey:"plugin.security.file.key" type:"path_string"`
-	FileSecurityCA          string `confkey:"plugin.security.file.ca" type:"path_string"`
-	FileSecurityCache       string `confkey:"plugin.security.file.cache" type:"path_string"`
+	FileSecurityCertificate string   `confkey:"plugin.security.file.certificate" type:"path_string"`
+	FileSecurityKey         string   `confkey:"plugin.security.file.key" type:"path_string"`
+	FileSecurityCA          string   `confkey:"plugin.security.file.ca" type:"path_string"`
+	FileSecurityCache       string   `confkey:"plugin.security.file.cache" type:"path_string"`
+
+	// TLS Parameters
+	CipherSuites            []string `confkey:"plugin.security.cipher_suites" type:"comma_split"`
```

Comment:
@ripienaar this is the only place I have to make this change, right?

Not working for me locally..

Reply:
that should be all yeah, but go-security will have to have this hash in its go mod